app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class homecontroller{ $th }
+
+class homecontrollerwa { $th }
+
+
+class HomeController extends Controller
```

Comment:
__[code_sniffer]__
* Each class must be in a file by itself

<!-- Comment by [SideCI](https://sideci.com) -->